modules/govuk/manifests/user.pp

```diff
@@ -68,9 +74,12 @@
     $membership = 'minimum'
   }
 
+  validate_re($expiry, '^(absent|\d{4}-\d{2}-\d{2})$')
```

Comment:
It'd be nice, but complex to implement ourselves and I don't think there's anything readily available that we could use.